localstack/utils/bootstrap.py

```diff
@@ -376,6 +379,246 @@ def extract_port_flags(user_flags, port_mappings: PortMappings):
+            )
+
+        return _cfg
+
+    @staticmethod
+    def container_name(name: str):
+        def _cfg(cfg: ContainerConfiguration):
+            cfg.name = name
+            cfg.env_vars["MAIN_CONTAINER_NAME"] = cfg.name
+
+        return _cfg
+
+    @staticmethod
+    def random_container_name(cfg: ContainerConfiguration):
+        cfg.name = f"localstack-{short_uid()}"
+        cfg.env_vars["MAIN_CONTAINER_NAME"] = cfg.name
+
+    @staticmethod
+    def default_container_name(cfg: ContainerConfiguration):
+        cfg.name = config.MAIN_CONTAINER_NAME
+        cfg.env_vars["MAIN_CONTAINER_NAME"] = cfg.name
+
+    @staticmethod
+    def service_port_range(cfg: ContainerConfiguration):
+        cfg.ports.add([config.EXTERNAL_SERVICE_PORTS_START, config.EXTERNAL_SERVICE_PORTS_END])
+        cfg.env_vars["EXTERNAL_SERVICE_PORTS_START"] = config.EXTERNAL_SERVICE_PORTS_START
+        cfg.env_vars["EXTERNAL_SERVICE_PORTS_END"] = config.EXTERNAL_SERVICE_PORTS_END
+
+    @staticmethod
+    def random_service_port_range(num: int = 50):
+        """
+        Tries to find a contiguous list of random ports on the host to map to the external service port
+        range in the container.
+        """
+
+        def _cfg(cfg: ContainerConfiguration):
+            port_range = get_free_tcp_port_range(num)
+            cfg.ports.add([port_range.start, port_range.end])
+            cfg.env_vars["EXTERNAL_SERVICE_PORTS_START"] = str(port_range.start)
+            cfg.env_vars["EXTERNAL_SERVICE_PORTS_END"] = str(port_range.end)
+
+        return _cfg
+
+    @staticmethod
+    def debug(cfg: ContainerConfiguration):
+        cfg.env_vars["DEBUG"] = "1"
+
+    @staticmethod
+    def network(network: str):
+        def _cfg(cfg: ContainerConfiguration):
+            cfg.network = network
+
+        return _cfg
+
+    @staticmethod
+    def custom_command(cmd: List[str]):
+        """
+        Overwrites the container command and unsets the default entrypoint.
+
+        :param cmd: the command to run in the container
+        :return: a configurator
+        """
+
+        def _cfg(cfg: ContainerConfiguration):
+            cfg.command = cmd
+            cfg.entrypoint = ""
+
+        return _cfg
+
+    @staticmethod
+    def env_vars(env_vars: Dict[str, str]):
+        def _cfg(cfg: ContainerConfiguration):
+            cfg.env_vars.update(env_vars)
+
+        return _cfg
+
+    @staticmethod
+    def port(*args, **kwargs):
+        def _cfg(cfg: ContainerConfiguration):
+            cfg.ports.add(*args, **kwargs)
+
+        return _cfg
+
+    @staticmethod
+    def volume(volume: VolumeBind):
+        def _cfg(cfg: ContainerConfiguration):
+            cfg.volumes.add(volume)
+
+        return _cfg
+
+    @staticmethod
+    def cli_params(params: Dict[str, Any]):
+        """
+        Parse docker CLI parameters and add them to the config. The currently known CLI params are:
+
+        :param params:
+        :return:
+        """
```

Comment:
It seems like something is missing here?

Reply:
My attention span when writing doc strings is very short